chains/ethereum/state_checks.go

```diff
@@ -0,0 +1,28 @@
+package ethereum
+
+import (
+	"math/big"
+
+	"github.com/ChainSafe/log15"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+)
+
+// Queries the contract for the current deposit status
+func (w *Writer) DepositStatus(depositId *big.Int, originChain *big.Int) (uint8, error) {
+	ret := new(struct {
+		Hash        [32]byte
+		Id          *big.Int
+		OriginChain *big.Int
+		NumYes      *big.Int
+		NumNo       *big.Int
+		Status      uint8
+	})
+	out := ret
```

Comment:
can we get rid of `ret` here?

Reply:
Code was copied from `contracts/Receiver/Receiver.go` 

Will modify